main.go

```diff
@@ -471,6 +478,39 @@ func handleCLIOptions(runOptions runOptionsType, env *uBootEnv, device *device,
 	return nil
 }
 
+// updateCheck sends a sigusr1 signal to the running mender daemon.
+func updateCheck() error {
+	cmd := exec.Command("pkill", "--signal", "SIGUSR1", "mender")
```

Comment:
We should use a better mechanism than this. `pkill` is dangerous to use in production software, because it can match many processes, even unrelated ones that happen to contain "mender" in their names. And it will signal the current process. And it might not even be there for restricted systems.

The classic approach is to use a pid file somewhere, but maybe an easier and more future oriented approach is to ask systemd which PID mender has. `systemctl status mender` shows it and I'm sure there is a way to get a more script-friendly variant of it (check the man page).